<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en">
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
	<meta name="keywords" content="Error Message, baan,baanerp,erp,forum,discussion,bulletin board" />
	<meta name="description" content="[Archive] Error Message Distribution, Transportation &amp; Warehousing" />
	
	<title>Error Message [Archive]  - Baanboard.com</title>
	<link rel="stylesheet" type="text/css" href="../styles.css" />
</head>
<body>
<div class="pagebody">
<div id="navbar"><a href="../index.html">Newbaanboard</a> &gt; <a href="../index.html">Baan Quick Support: Functional &amp; Technical</a> &gt; <a href="../index.html">Distribution, Transportation &amp; Warehousing</a> &gt; Error Message</div>
<hr />
<div class="pda"></div>

<hr />

<div class="post"><div class="posttop"><div class="username">django25</div><div class="date">25th February 2009, 07:34</div></div><div class="posttext">Sir/Madam<br />
<br />
We are getting the following message in Maintain Inventory Transactions by lot &amp; Location session – tdilc1120m000.<br />
”Location contains blocked goods. Continue?”<br />
<br />
However when we see the Display locations session for Warehouse – VS0, it contains no location.<br />
Maintain Items by Warehouse – tdinv0101m000 details for WH VS0 are as follows.<br />
Inventory on hand = 2<br />
Inv on hold = 2<br />
Allocated Inv = 2<br />
Rest all value is 0.<br />
<br />
Regards<br />
Django</div></div><hr />


<div class="post"><div class="posttop"><div class="username">avpatil</div><div class="date">5th March 2009, 20:48</div></div><div class="posttext">Inventory on hold means that 2 units are blocked.<br />
<br />
Arvind Patil</div></div><hr />



</div>
</body>
</html>